Gems/AWSMetrics/Code/Tests/MetricsManagerTest.cpp

```diff
@@ -471,7 +471,7 @@ namespace AWSMetrics
         ASSERT_EQ(m_metricsManager->GetNumBufferedMetrics(), MaxNumMetricsEvents / 2);
     }
 
-    TEST_F(MetricsManagerTest, OnResponseReceived_NoResponseRecords_RetryAllMetrics)
+    TEST_F(MetricsManagerTest, OnResponseReceived_NoresponseEntries_RetryAllMetrics)
```

Comment:
nit: should be capital letter in "...No**R**esponseEntries_..." ?

Reply:
Sure. I'll make the change.